src/action.js

```diff
@@ -242,7 +243,11 @@ define(function (require) {
                 return;
             }
 
+
             url = resolveUrl(url);
+
+            reuseService(url, data);
```

Comment:
```suggestion
            useService(url, data);
```
感觉用 use 比较好，看起来没有 re；另外是不是直接把 data.service 传进去？

Reply:
ok